gammapy/maps/geom.py

```diff
@@ -593,6 +593,24 @@ def slice(self, idx):
             unit=self._unit,
         )
 
+    def squash(self):
+        """Create a new axis object by squashing the axis into one bin.
+
+        Returns
+        -------
+        axis : `~MapAxis`
+            Sliced axis objected.
```

Comment:
Please adapt the docstring here and fix the typo.

Reply:
Done. Also fixed typo in preceeding function